CHANGELOG.next.toml

```diff
@@ -171,3 +171,15 @@ message = "Update MSRV to Rust 1.67.1"
 references = ["smithy-rs#2611"]
 meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "all"}
 author = "jdisanti"
+
+[[aws-sdk-rust]]
+message = "Refactor aws-smithy-types crate. Blob, Datetime, Number and Document structs now goes to it's own files. No breaking changes."
+references = ["smithy-rs#2616"]
+meta = { "breaking" = false, "tada" = false, "bug" = true }
+author = "thomas-k-cameron"
+
+[[smithy-rs]]
+message = "Refactor aws-smithy-types crate. Blob, Datetime, Number and Document structs now goes to it's own files. No breaking changes."
+references = ["smithy-rs#2616"]
+meta = { "breaking" = false, "tada" = false, "bug" = true }
+author = "thomas-k-cameron"
```

Comment:
These changes don't change the public API in any way, right? If so, I don't think we need these changelog entries. I'm also not sure about why the `"bug" = true` either 🤔